### DISCUSSION TITLE
Change config remotely

### DISCUSSION THREAD

**Discussion by 9mateusz** [awawa-dev/HyperHDR#438] [Q&A]
Hi,
Is there any way to change config in different way than from web? 
I would like to prepare several configs (mainly different led layout, led overlap) and change them somehow. Maybe via json, Maybe link, Maybe via sending command to rpi to activate cfg from another file.
For movies I prefere bigger led overlap to make everything smooth, but for sports/games without overlap. (i know smoothong option, but it is not solution here) 
What is important for me, is possibility of automation of this proces, to update config by single click.
Best solution would be to change whole config, but also changing led layout only would be satisfying.

**Comment by awawa-dev** [comment]
Hi
Theoretically there is a such solution, 'load-db' command in the JSON API https://github.com/awawa-dev/HyperHDR/blob/75762ca23f45d9531cf82f28bc2f60c3ef01d13b/sources/api/JsonAPI.cpp#L205 It's used to restore full json configuration that you can make on the 'general' tab. For systemd OS daemon also restarts HyperHDR (on other OS HyperHDR just stops). But it requires to acquire admin privileges first.

**Reply by 9mateusz** [reply]
is there any way to store several configs on server (RPI in my case) and then execute some command to use selected?

**Reply by awawa-dev** [reply]
You can pass `userdata` parameter when starting HyperHDR application, to select a folder that contains HyperHDR's custom configuration.
